contrib/debian/netdata-plugin-ebpf.postinst

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -e
+
+case "$1" in
+  configure|recnfigure)
```

Comment:
Is `recnfigure` a typo error? Or is this intentional?

Reply:
Nice catch, that is indeed a typo.